tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -2314,10 +2268,10 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
     # create expectation suite
     context.add_expectation_suite("my_expectation_suite")
 
-    batch_request: FluentBatchRequest = FluentBatchRequest(
-        datasource_name="my_sqlite_datasource",
-        data_asset_name="table_partitioned_by_date_column__A_query_asset_limit_5",
-    )
+    batch_request = {
```

Comment:
This is the correct change, due to the type requirements in the `Checkpoint` configuration.  There is a JIRA ticket to re-examine the `Checkpoint` design, including proposing emphasis on typed objects and formal interfaces.